In-house patch to enable Solaris Zones specific vm_mode value for use by the
ImagePropertiesFilter scheduling filter.  This patch has not yet been
submitted upstream.

--- nova-2015.1.2/nova/compute/vm_mode.py.~1~	2015-10-13 07:52:35.000000000 -0700
+++ nova-2015.1.2/nova/compute/vm_mode.py	2015-12-31 20:11:22.020900882 -0800
@@ -31,8 +31,9 @@ HVM = "hvm"  # Native ABI (aka fully vir
 XEN = "xen"  # Xen 3.0 paravirtualized
 UML = "uml"  # User Mode Linux paravirtualized
 EXE = "exe"  # Executables in containers
+SOL = "solariszones"  # Solaris zones (non-global and kernel)
 
-ALL = [HVM, XEN, UML, EXE]
+ALL = [HVM, XEN, UML, EXE, SOL]
 
 
 def get_from_instance(instance):
